### DISCUSSION TITLE
Aren't symbolic links enabled by default nowadays?

### DISCUSSION THREAD

**Discussion by theking2** [git-for-windows/git#3994] [Ideas]
`fsutil behavior query SymlinkEvaluation`

gives me 
```
Local-to-local symbolic link evaluation is: ENABLED
Local-to-remote symbolic link evaluation is: ENABLED
Remote-to-local symbolic link evaluation is: DISABLED
Remote-to-remote symbolic link evaluation is: DISABLED
```

Symlinks are enabled by default nowadays, aren't they. Why not reflect this in the installer?

**Comment by dscho** [comment]
> Symlinks are enabled by default nowadays, aren't they.

Nope. [Only in Developer Mode, and only when running Windows 10 build 14972 or later](https://blogs.windows.com/windowsdeveloper/2016/12/02/symlinks-windows-10/).

But Git for Windows still supports Windows versions all the way back to [Windows 7 SP 1](https://gitforwindows.org/requirements.html).

> Why not reflect this in the installer?

[It is reflected in the installer](https://github.com/git-for-windows/build-extra/blob/0471d36424fc9bfe15560c8623eb05174c8e1226/installer/install.iss#L1247-L1275).
